### PR TITLE
chore: write Changelog

### PR DESCRIPTION
**Pull request by salisdemos**
Add changes to changelog hence they have been forgotten

### CODE REVIEW

**Review by elmasdemos — APPROVED** [review]
thanks! lgtm, code review

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+- Fix: Move Translator out of template
```
Comment (hwiem) [inline CHANGELOG.md:5]: I find this too unspecific. Which template? Also: Why? What does it fix? (Of course we don't want to go into too much detail here, but a little context would be helpful)
I think it's a good idea to include the type (fix, chore, etc.) here, but we should agree on it and then use it consistently.

**Review by hwiem — CHANGES_REQUESTED** [review]

**Comment by hwiem** [comment]
Side note: Could you reference the PR(s) this changelog change refers to in the PR description? :pray:

File: CHANGELOG.md (hunk 1 above)
Comment (spiess-demos) [reply CHANGELOG.md:5]: I think there is no need in mentioning any implementation detail. If this fixes a bug, it can be mentioned here.

Comment (hwiem) [reply CHANGELOG.md:5]: Yeah, something like "fix `undefined Translator` error in `componentName`" would do

Comment (salisdemos) [reply CHANGELOG.md]: True. changed in 97506fe0ffcecee2fe83ea1af4a4ebed56667983

**Review by spiess-demos — APPROVED** [review]
Lgtm!

**Review by hwiem — APPROVED** [review]

**Pull request merged**